### PR TITLE
Add f17 bt params

### PR DESCRIPTION
**Pull request by sc0tts**
This PR adds F17 parameters for both the Bootstrap and the NASATeam algorithms.

### CODE REVIEW

File: pm_icecon/bt/params/nsidc0001.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    weather_filter_seasons=NSIDC0001_F17_SOUTH_PARAMS["weather_filter_seasons"],
+)
+
+
+CDR_F17_NORTH_PARAMS = NSIDC0001_F17_NORTH_PARAMS.copy()
+
+CDR_F17_SOUTH_PARAMS = NSIDC0001_F17_SOUTH_PARAMS.copy()
+
+
+def get_F17_params(
```
Comment (trey-stafford) [inline pm_icecon/bt/params/nsidc0001.py:150]: Do we end up using both this and `get_F17_bootstrap_params`?

**Review by trey-stafford — CHANGES_REQUESTED** [review]
Needs some cleanup of commented-out code, otherwise looks OK. I think we still need to do some work to streamline how params are handled. Its confusing that we have multiple copies of params in different formats. E.g., 

`NSIDC0001_F17_NORTH_PARAMS`, `BOOTSTRAP_PARAMS_INITIAL_F17_NORTH`, `CDR_F17_NORTH_PARAMS` in `bt/params/nsidc0001.py`.

**Comment by trey-stafford** [comment]
We should also add an entry to the CHANGELOG!

File: pm_icecon/bt/params/nsidc0001.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+from pm_icecon.bt.params.util import setup_bootstrap_params_dict
+from pm_icecon.config.models.bt import (
+    BootstrapParams,
+    TbSetParams,
+    WeatherFilterParams,
+    WeatherFilterParamsForSeason,
+)
+from pm_icecon.gridid import get_gridid_hemisphere
+
+# GODDARD_AMSR2_NORTH_PARAMS = ParamsDict(
```
[the remark this answers is not in the document]
Comment (sc0tts) [reply pm_icecon/bt/params/nsidc0001.py:23]: Yep, that was from dev work.  
The no-longer-useful commented-out lines have been removed.

File: pm_icecon/bt/params/ausi_amsr2.py [hunk 3]
```diff
@@ -183,6 +183,7 @@ def get_ausi_bootstrap_params(
 ) -> dict:
     """Assign the bootstrap parameters for this date, sat, grid."""
     hemisphere = get_gridid_hemisphere(gridid)
+    # TODO: Need to add a "ame" conditional here
```
[the remark this answers is not in the document]
Comment (sc0tts) [reply pm_icecon/bt/params/ausi_amsr2.py]: I've split up the AMSRE and the AMSR2 parameter codes.

File: pm_icecon/bt/params/nsidc0001.py [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+                # wxlimt=18.596,
+                wintrc=93.2861,
+                wslope=0.497374,
+                wxlimt=16.5,
+            ),
+        ),
+    ],
+)
+
+# TODO: Is this where we would overwrite the default values?
```
[the remark this answers is not in the document]
Comment (sc0tts) [reply pm_icecon/bt/params/nsidc0001.py:91]: The TODO is meant to indicate that part of the complexity of this code is that we want to show that the values we use for the CDR are (1) based on the values that are used for the AU_SI12 bootstrap code, but are also (2) overwritten in part by values that we calculated while implementing the algorithm for eCDR.

**Comment by trey-stafford** [comment]
This PR needs to wait on the release of `pm_tb_data`  v0.3.0.

**Comment by trey-stafford** [comment]
Declining in favor of https://github.com/nsidc/pm_icecon/pull/45 - better branch name.

**Pull request closed without merging**